src/config/dynamic-theme-fixes.config

```diff
@@ -7557,6 +7557,23 @@ INVERT
 .serviceLogoImg
 .serviceIcon
 
+===============================
+
+kraken.com
+
+CSS
+header > nav > ul > li > a::after, ._3ku3YkWZbOVnguKutXHfK- {
+    background-color: var(--darkreader-neutral-text) !important;
```

Comment:
`._3ku3YkWZbOVnguKutXHfK-` what is that selector used for?

Reply:
It's the 'price gauge', I don't see other way to get it

![image](https://user-images.githubusercontent.com/26639800/140540255-8cd26742-912e-457e-8a2b-d6a90df59d81.png)

Reply:
```css
span[data-testid^=asset-pair"][data-testid$="-low"] + div > :last-child
```

Reply:
😮  great, thanks!